RELEASE.md

```diff
@@ -5,6 +5,12 @@ Please follow the established format:
 - Use present tense (e.g. 'Add new feature')
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
+# Release 8.1.0
+
+## Major features and improvements
+
+- Extending stateful URLs with node types and tags. (#1799) 
```

Comment:
[Nit] I think the PR also includes expand/collapse modular pipeline. So the text can be something like - `Extending stateful URLs with node filters and expand/collapse modular pipelines`